TARS/src/mcu_main/sensors/VoltageSensor.h

```diff
@@ -4,67 +4,28 @@
 #include "ChRt.h"
```

Comment:
Should we have guards for `ENABLE_VOLTAGE`?

Reply:
Can you go ahead and add that? It makes sense now since we directly read voltage from the Teensy instead of reading it via serial from another MCU